clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -1034,44 +1034,44 @@ check_branch_decoration()
         std::vector<memref_t> memrefs = {
             gen_marker(TID, TRACE_MARKER_TYPE_VERSION, TRACE_ENTRY_VERSION_BRANCH_INFO),
             gen_instr(TID, /*pc=*/1),
-            gen_marker(TID, TRACE_MARKER_TYPE_BRANCH_TARGET, 32),
-            gen_instr_type(TRACE_TYPE_INSTR_INDIRECT_CALL, TID, /*pc=*/2),
+            gen_instr_type(TRACE_TYPE_INSTR_INDIRECT_CALL, TID, /*pc=*/2, /*size=*/1,
+                           /*target=*/32),
             gen_marker(TID, TRACE_MARKER_TYPE_KERNEL_EVENT, 32),
             gen_instr(TID, /*pc=*/999),
         };
         if (!run_checker(memrefs, false))
             return false;
     }
 #endif
-    // Indirect branch target: no marker.
+    // Indirect branch target: incorrect zero target PC.
     {
         std::vector<memref_t> memrefs = {
             gen_marker(TID, TRACE_MARKER_TYPE_VERSION, TRACE_ENTRY_VERSION_BRANCH_INFO),
             gen_instr(TID, /*pc=*/1),
-            gen_instr_type(TRACE_TYPE_INSTR_INDIRECT_CALL, TID, /*pc=*/2),
-            gen_instr(TID, /*pc=*/32),
+            gen_instr_type(TRACE_TYPE_INSTR_INDIRECT_CALL, TID, /*pc=*/2, /*size=*/1,
+                           /*target=*/0),
         };
         if (!run_checker(memrefs, true,
-                         { "Indirect branches must be preceded by their targets", TID,
+                         { "Indirect branches must contain targets", TID,
                            /*ref_ordinal=*/3, /*last_timestamp=*/0,
                            /*instrs_since_last_timestamp=*/2 },
-                         "Failed to catch missing indirect branch target marker"))
+                         "Failed to catch missing indirect branch target field"))
             return false;
     }
-    // Indirect branch target: marker value incorrect.
+    // Indirect branch target: incorrect target value.
```

Comment:
Should we call this an incorrect target value or a PC disc?

Reply:
It seems a different thing from a discontinuity b/c the next PC in the trace seems like the "real" one.